docs/users/update_basic_user_data.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Update basic user account data from the console.
+---
+
+# Update basic user data from CLI
+
+Multiple user management scenarios may result in having to update basic user account data, such as user status, the password, or email.
+Especially, when a security breach occurs, you may need to revoke user access by disabling the account, or change the user's password.
+
+You can do it without accessing the Admin UI, by running a console command.
+You reference the user account with either user id or login.
+
+## Disable or enable user account
+
+To change the status of a user account, you must have the `user/activation` permission.
+
+Disable the user account:
+
+```bash
+php bin/console ibexa:user:update-user --disable <user_reference>
```

Comment:
Here and in other places:
```suggestion
php bin/console ibexa:user:update-user --disable <login>
```
so the usage might look like:
`php bin/console ibexa:user:update-user --disable admin`